src/constants/pools.ts

```diff
@@ -670,3 +670,12 @@ const POOLS_MAP = {
 export const POOLS: Pools = POOLS_MAP[networkId.value]
   ? POOLS_MAP[networkId.value]
   : POOLS_GENERIC;
+
+export const ALLOWED_PRICE_RATE_PROVIDERS = {
+  '0xa13a9247ea42d743238089903570127dda72fe44': {
+    '0x0000000000000000000000000000000000000000': true,
```

Comment:
Thanks, I changed it so that it applies to all tokens.

For the rest of the providers, do you think I should get a list from somewhere? Check what's already in subgraph? Or leave it as it is?

Reply:
Thank you. I also just realized this is one list for all networks, when it should be network specific like the pools allowlists to avoid collision issues. At which point maybe this belongs in it own file, like `constants/rateProviders.ts`?

Reply:
And yes, for the rest of the providers we can start with copying all from the subgraphs - there aren't many, so it should be easy to review them one by one to make sure they're sound. The Maxis could probably help with this review too. cc @Xeonus @zekraken-bot

Reply:
Shall I use this query to get the lists by network?

https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2/graphql?query=query+%7B%0A++pools%28filter%3A+%7B%0A++++type%3A+%22Weighted%22%0A++%7D%29+%7B%0A++++priceRateProviders+%7B%0A++++++address%0A++++++token+%7B%0A++++++++address%0A++++++%7D%0A++++%7D%0A++%7D%0A%7D%0A

Reply:
That works. [This](https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2/graphql?query=query+%7B%0A++priceRateProviders+%28%0A++++where%3A+%7B%0A++++++address_not%3A%220x0000000000000000000000000000000000000000%22%0A++++++poolId_%3A%7B%0A++++++++poolType%3A%22Weighted%22%0A++++++%7D%0A++++%7D%0A++%29%0A++++%7B%0A++++token+%7B%0A++++++address%0A++++%7D%0A++++address%0A++%7D%0A%7D%0A) might be easier to parse

Reply:
Ok, values added.